tests/test_nautilus_librarian/test_typer/test_commands/test_workflows/test_actions/test_auto_commit_images.py

```diff
@@ -58,6 +58,43 @@ def test_calculate_the_corresponding_base_image_from_gold_image():
     )
 
 
+def remove_base_image_dvc_files(temp_git_dir):
+    os.remove(f"{temp_git_dir}/data/000001/42/000001-42.600.2.tif.dvc")
+    os.remove(f"{temp_git_dir}/data/000001/42/.gitignore")
+
+
+def commit_added_base_images(temp_git_dir, temp_gpg_home_dir, git_user):
+
+    dvc_diff = {
+        "added": [
+            {"path": "data/000001/32/000001-32.600.2.tif"},
+        ],
+        "deleted": [],
+        "modified": [],
+        "renamed": [],
+    }
+
+    return auto_commit_base_images(
+        compact_json(dvc_diff), str(temp_git_dir), str(temp_gpg_home_dir), git_user
+    )
+
+
+def check_commit_assertions(
```

Comment:
hi @yeraydavidrodriguez 

1. Refactor name? `assert_commit_content_and_committer(expected_git_user, git_user, expected_signingkey, signingkey)`
2. Break it down into 2 asserts: `assert_commit_content()` and `asser_commit_signingkey`
3. Continue using assert: `assert committer_are_equal(expected_committer)` and `assert assert signingkey_are_equal(expected_signing_key, signing_key)`
4. Use other libraries like https://github.com/hamcrest/PyHamcrest

Regardless we change the assert standard format (`assert {expr}`) or not I think we are asserting two things at the same time. If we want to have a high-level commit assertion I think I would do something like:

Only for committer:

```

expected_committer = new GitCommitter(name, email, signingkey)
expected_committer = GitCommitter.fromGitUser(git_user)

committer = extract_committer(newly created commit) <- it returns a GitCommiter

...

assert expected_committer == comitter
```

For commits content, we need to repeat the same or use the object provided by GitPython. We could also create a `GitCommit` with both the committer info and the commit contents.

We can discuss this in the future, but I think it makes sense to define our custom test scaffolding conventions.